packages/site-content/package.json

```diff
@@ -15,7 +15,8 @@
       "command": "eleventy --config=eleventy.config.prod.cjs",
       "files": [
         "eleventy.config.prod.cjs",
-        "site"
+        "site",
+        "../site-server/src/templates/base.ts"
```

Comment:
Maybe `../site-server/lib/templates/**/*.js` so that we don't need to enumerate if we add more?